src/components/nodes/SwapNode.tsx

```diff
@@ -9,16 +17,36 @@ export function SwapNode({
   selected: boolean;
   data: ISwapData;
 }) {
+  const { fetchBalance } = useSafeBalances();
+  const { getOrderDataByOrderId } = useRecipeData();
+  const recipeData = getOrderDataByOrderId(data.orderId);
+  if (!recipeData) return null;
+  const sellTokenWalletAmount = Number(
+    formatUnits(
+      BigInt(fetchBalance(data.tokenSell.address)),
+      data.tokenSell.decimals
+    )
+  );
+  const [sellAmount, buyAmount] = calculateAmounts(recipeData);
+  const sellAmountWithSymbol = `${formatNumber(sellAmount, 2, "decimal", "compact", 0.01)} ${data.tokenSell.symbol}`;
+  const buyAmountWithSymbol = `${formatNumber(buyAmount, 2, "decimal", "compact", 0.01)} ${data.tokenBuy.symbol}`;
   return (
     <BaseNode selected={selected}>
-      <div className="flex">
-        <div className="ml-2">
-          <div className="text-sm font-bold text-highlight">Swap</div>
-          <div className="text-xs">
-            {`${data.isSellOrder ? "Sell" : "Buy"} ${data.amount} ${
-              data.tokenSell.symbol
-            } for ${data.tokenBuy.symbol}`}
-          </div>
+      <div className="flex flex-col">
+        <div className="flex flex-row gap-2 items-center">
+          <span className="text-sm font-bold text-highlight">Swap</span>
+          {sellTokenWalletAmount < sellAmount && (
+            <a>
+              <Tooltip content="You don't have enough amount of the selling token. The order can still be posted but it just can be filled when you have the tokens.">
```

Comment:
just trying to make it clear, but not sure if my rewrite is 100% accurate
```suggestion
              <Tooltip content="You don't have enough amount of the selling token. The order can still be posted but it be filled immediately whenever you have enough tokens in your wallet.">
```

Reply:
Actually, it isn't immediately but if it you don't have the tokens the order will never be filled. Vou mudar para:

`"You don't have enough amount of the selling token. The order can still be posted but it will never be filled unti you don't have enough tokens in your wallet."`